p2p/transport/quic/transport.go

```diff
@@ -166,19 +299,10 @@ func NewTransport(key ic.PrivKey, psk pnet.PSK, gater connmgr.ConnectionGater, r
 	}
 	config.Tracer = tracer
 
-	tr := &transport{
-		privKey:      key,
-		localPeer:    localPeer,
-		identity:     identity,
-		connManager:  connManager,
-		gater:        gater,
-		rcmgr:        rcmgr,
-		conns:        make(map[quic.Connection]*conn),
-		holePunching: make(map[holePunchKey]*activeHolePunch),
-	}
 	config.AllowConnectionWindowIncrease = tr.allowWindowIncrease
 	tr.serverConfig = config
 	tr.clientConfig = config.Clone()
+	tr.clientConfig.AcceptToken = new(accept).setOptions(tr.period, tr.fraction).acceptFunction
```

Comment:
This needs to be set on the server config, not on the client config ;)

Reply:
I did it also on the server but here there was a call to Clone(), i don't know how it's implemented but I assumed that it uses the same AccessToken instance, thus causing a conflict. Isn't that the behaviour?

Reply:
Oh, I missed that earlier part. It's probably better to clone a server config and a client config in L288, and then only set the callback on the server config.